java/HealthVault Mobile/HVSample/src/main/res/layout/weight.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This layout can be optimized a lot. We should remove unneeded nesting of containers and revisit